### ISSUE TITLE
"no-globals" rule is also used for xml-views

### ISSUE THREAD

**Issue by FabianBerr** [SAP/ui5-linter#952]
## Expected Behavior

ui5lint does not care about "no-globals" in xml-views of ui5 application
**OR** 
To have the ability to exclude some rules for specific file types

## Current Behavior

it does throw an error when using type formatting in xml views
```xml
 <Text text="{path : 'SOURCEBESTBEFOREDATE', type : 'sap.ui.model.type.Date', formatOptions: { style : 'medium'}}" wrapping="false" /> />
```

leads to:
```
error Access of global variable 'sap' (sap.ui.model.type.Date)  no-globals
```

## Steps to Reproduce the Issue

Use XML snippet above

## Context

- UI5 linter version: **`1.20.5`**
- Node.js version: **`v20.19.0`**
- npm version: `10.8.2`
- OS/Platform: `Linux/BAS`
- Browser *(if relevant)*: `Edge`

## Log Output / Stack Trace

--

**Comment by flovogt** [comment]
I know the guide could be improved but at least it mentiones this in [Best Practices for Developers](https://ui5.sap.com/#/topic/28fcd55b04654977b63dacbee0552712)

> Don't use global names in your XML. Ensure that the target function or object is defined as a module and require the defined module via [core:require in the XML](https://ui5.sap.com/topic/b11d853a8e784db6b2d210ef57b0f7d7). Use `template:require` if the XML content needs preprocessing.

**Comment by FabianBerr** [comment]
Thank you for the hint :-)

was not aware of core:require..

It worked, thank you

```xml
<core:FragmentDefinition 
...
xmlns:core="sap.ui.core"
core:require="{ Date: 'sap/ui/model/type/Date' }">
```
```xml
<Text text="{path : 'SOURCEBESTBEFOREDATE', type : 'Date', formatOptions: { style : 'medium'}}" wrapping="false" />
```
